aspnetcore/blazor/hybrid/routing.md

```diff
@@ -15,10 +15,27 @@ This article explains how to manage request routing and navigation in Blazor Hyb
 
 [!INCLUDE[](~/blazor/includes/blazor-hybrid-preview-notice.md)]
 
-Register to the `ExternalNavigationStarting` event and set the `ExternalLinkNavigationEventArgs.ExternalLinkNavigationPolicy` property to change link handling behavior. The `ExternalLinkNavigationPolicy` enumeration (`enum`) allows setting link handling behavior to `OpenInExternalBrowser`, `InsecureOpenInWebView`, and `CancelNavigation`. The `ExternalLinkNavigationEventArgs.Uri` property can be used to dynamically set link handling behavior.
+[!INCLUDE[](~/blazor/includes/net-maui-release-candidate-notice.md)]
+
+Default URI request routing behavior:
+
+* A link is *internal* if the host name and scheme match between the app's origin URI and the request URI. When the host names and schemes don't match or if the link sets `target="_blank"`, the link is considered *external*.
+* If the link is internal, the link is opened in the `BlazorWebView` by the app.
+* If the link is external, the link is opened by an app determined by the device based on the device's registered handler for the link's scheme.
+* For internal links that appear to request a file because the last segment of the URI uses dot notation (for example, `/file.x`, `/Maryia.Melnyk`, `/image.gif`) but don't point to any static content:
+  * WPF and Windows Forms: The host page content will be returned.
+  * .NET MAUI: A 404 response will be returned.
```

Comment:
Since this is an edge case that will be rarely encountered by customers, I almost wonder if it would be better to put this in a "WARNING" box instead? We would warn specifically about using file-like page URIs on MAUI platforms.

Reply:
Fine with me either way ... whatever you think is best.

Reply:
This should be present tense tho, so I'll make that change at the end if you don't change it when making a WARNING.

* The host page content will be returned. 👉 The host page content is returned.
* A 404 response will be returned. 👉 A 404 response is returned.